libs/core/src/components/sage-checkbox/test/sage-checkbox.e2e.ts

```diff
@@ -0,0 +1,148 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('sage-checkbox', () => {
+  it('renders', async () => {
```

Comment:
These tests are specs in that they are testing whether are not something is there. You already have those in the `spec` test. Those are the ones giving you the 100 percent coverage. These are more like toggling state. They mostly involve some type of event occurring. 

One example can be to switch from not checked -> indeterminate -> checked and check stuff along the way or flows similar to that in your component.